_resources/port1.0/group/aspelldict-1.0.tcl

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+#
+# This PortGroup automatically sets all the fields of the various aspell
+# directories ports (e.g. aspell-dict-uk).
+#
+# Usage:
+#
+#   PortGroup               aspelldict 1.0
+#   aspelldict.setup        locale version lang aspell_version
+#
+# You can get ${aspell_version} from ${distname}. For example in
+# "aspell6-uk-1.4.0-0.tar.bz2" is 6. You can find the list of all dictionaries
+# here <https://ftp.gnu.org/gnu/aspell/dict/0index.html>
+#
+# Example:
+#
+#   PortGroup               aspelldict 1.0
+#   aspelldict.setup        uk 1.4.0-0 {Ukrainian} 6
+
+proc aspelldict.setup {locale version lang {aspell-version {}}} {
+    global description prefix
+
+    name                 aspell-dict-${locale}
+    version              ${version}
+    categories           textproc
+    platforms            darwin
```

Comment:
this is already the default and doesn't need to be set anymore

Reply:
Thank you, I forgot that platforms are already by default